webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -89,7 +71,20 @@ export default {
   data: () => ({
     filterToChange: null,
     bottomMenu: false,
+    startSearchAfterInMilliseconds: 300,
+    endTypingKeywordTimeout: 50,
+    startTypingKeywordTimeout: 0,
+    typing: false,
+    advancedFilterCount: 0,
+    showMobileFilter: false,
```

Comment:
You inverted the best practices of variables and methods naming conventions. The verb should be used for the method `mobileFilter()` => `showMobileFilter()` and the variable has to be modified to anoun or adjective from `showMobileFilter` => `mobileFilter`

```suggestion
    mobileFilter: false,
```

Reply:
fixed